pkgs/development/python-modules/django-postgresql-netfields/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, buildPythonPackage
+, django
+, netaddr
+, six
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "django-postgresql-netfields";
+
+  src = fetchFromGitHub {
+    owner = "jimfunk";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "1rrh38f3zl3jk5ijs6g75dxxvxygf4lczbgc7ahrgzf58g4a48lm";
+  };
+
+  # Failing because of ENV variables
```

Comment:
you can set variables like so:
```
  checkPhase = ''
    HOME=$TMPDIR SOME_VAL=FOO pytest
  '';
```

Reply:
Well, I managed to run tests, but looks like it requires connection to a postgres database. Kept the changes as comment for future reference, if we can somehow supply that later.